src/net.cpp

```diff
@@ -1031,6 +1060,9 @@ void ThreadSocketHandler()
             BOOST_FOREACH(CNode* pnode, vNodesCopy)
                 pnode->Release();
         }
+
+        if (progress == 0)  // Nothing happened even though select did not block.  So slow us down.
+            MilliSleep(50);
```

Comment:
I'm not sure I see the rational for the Millisleep(50) here.   Select
will block until it's timeout is reached.  If there's nothing to do it
will just go back to select.  Why the need for an additional sleep? 


On 14/10/2015 11:01 AM, Mike Hearn wrote:
>
> In src/net.cpp
> <https://github.com/bitcoinxt/bitcoinxt/pull/81#discussion_r42028933>:
>
> > @@ -1031,6 +1060,9 @@ void ThreadSocketHandler()
> >              BOOST_FOREACH(CNode* pnode, vNodesCopy)
> >                  pnode->Release();
> >          }
> > +
> > +        if (progress == 0)  // Nothing happened even though select did not block.  So slow us down.
> > +            MilliSleep(50);
>
> It'd be good to have a rationale in the comments for why 50 msec here.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/bitcoinxt/bitcoinxt/pull/81/files#r42028933>.
>